The-SMIRNOFF-force-field-format.md

```diff
@@ -304,31 +312,36 @@ Support for [other Lennard-Jones mixing schemes](https://en.wikipedia.org/wiki/C
 
 Later revisions will add support for additional potential types (e.g., `Buckingham-exp-6`), as well as the ability to support arbitrary algebraic functional forms using a scheme such as
 ```XML
-<vdW potential="4*epsilon*((sigma/r)^12-(sigma/r)^6)" scale12="0.0" scale13="0.0" scale14="0.5" scale15="1" sigma_unit="angstrom" epsilon_unit="kilocalories_per_mole" switch_width="8.0" switch_width_unit="angstrom" cutoff="9.0" cutoff_unit="angstrom" long_range_dispersion="isotropic">
+<vdW potential="4*epsilon*((sigma/r)^12-(sigma/r)^6)" scale12="0.0" scale13="0.0" scale14="0.5" scale15="1" switch_width="8.0*angstrom" cutoff="9.0*angstrom" long_range_dispersion="isotropic">
    <CombiningRules>
       <CombiningRule parameter="sigma" function="(sigma1+sigma2)/2"/>
       <CombiningRule parameter="epsilon" function="sqrt(epsilon1*epsilon2)"/>
    </CombiningRules>
-   <Atom smirks="[#1:1]" sigma="1.4870" epsilon="0.0157"/>
-   <Atom smirks="[#1:1]-[#6]" sigma="1.4870" epsilon="0.0157"/>
+   <Atom smirks="[#1:1]" sigma="1.4870*angstrom" epsilon="0.0157*kilocalories_per_mole"/>
+   <Atom smirks="[#1:1]-[#6]" sigma="1.4870*angstrom" epsilon="0.0157*kilocalories_per_mole"/>
    ...
 </vdW>
 ```
 If the `<CombiningRules>` tag is provided, it overrides the `combining_rules` attribute.
 
 Later revisions will also provide support for special interactions using the `<AtomPair>` tag:
 ```XML
-<vdW potential="Lennard-Jones-12-6" combining_rules="Lorentz-Berthelot" scale12="0.0" scale13="0.0" scale14="0.5" sigma_unit="angstroms" epsilon_unit="kilocalories_per_mole">
-   <AtomPair smirks1="[#1:1]" smirks2="[#6:2]" sigma="1.4870" epsilon="0.0157"/>   
+<vdW potential="Lennard-Jones-12-6" combining_rules="Lorentz-Berthelot" scale12="0.0" scale13="0.0" scale14="0.5" scale15="1">
+   <AtomPair smirks1="[#1:1]" smirks2="[#6:2]" sigma="1.4870*angstrom" epsilon="0.0157*kilocalories_per_mole"/>   
    ...
 </vdW>   
 ```
 
+| vdW section tag version | Required tag attributes and default values                                                                                                                                                                                        | Optional tag attributes | Required parameter attributes                 | Optional parameter attributes |
```

Comment:
I'm probably missing something, but why do we have default values with required attributes?

Reply:
I now realize that there are no "optional tag attributes" for any parameter section tags. I'm going to remove that column from all tables, and change this to "tag attributes and default values"